Source/ChilliSource/Core/Scene/Scene.h

```diff
@@ -53,10 +54,10 @@ namespace ChilliSource
         ///
         /// @author S Downie
         ///
-        /// @param Input system used by the window to listen
-        /// for input events
+        /// @param renderTarget
+        ///     The target into which the scene renders, if null renders to screen
         //-------------------------------------------------------
-        static SceneUPtr Create();
+        static SceneUPtr Create(TargetGroupUPtr renderTarget = nullptr);
```

Comment:
Probably best to avoid defaulting. The only time the user will be calling this is if they are using RTT so a target is required. As the user will be creating this through the CreateSystem<>() interface--and therefore aren't calling this directly and don't see the parameters--it's useful to get a compiler error if they don't supply parameters to make it easier to see what they've done wrong